### ISSUE TITLE
Error running bootstrap executable created by py2exe with Python 3.4

### ISSUE THREAD

**Issue by david-hughes** [cloudmatrix/esky#148]
Fatal Python error: Py_Initialize: unable to load the file system codec
ImportError: No module named 'encodings'

This appears to be identical to Issue #72 that was closed as a result of fixing #71 - although that one was concerned with errors in **creating** the executable. #104 seems more relevant with a similar problem using py2app where work was done to ensure that the relevant modules were made accessible from the bootstrap executable.

To recap, the bootstrap executable produces the above error while the one inside appdata runs OK.

esky (0.9.10.dev0)
py2exe (0.9.2.2)
Python 3.4
Windows 10 64 bit

**Comment by david-hughes** [comment]
Sorry, I just discovered #131 raised last January that describes the exact same issue. What's the situation regarding the development and support of Esky at the moment?

**Comment by rfk** [comment]
Thanks for reaching out.  Unfortunately this project is no longer actively maintained, so I'm going to move it into archive mode:

  https://rfk.id.au/blog/entry/archiving-open-source-projects/
